core1_0/buffer_copy.go

```diff
@@ -6,9 +6,10 @@ package core1_0
 */
 import "C"
 import (
-	"github.com/CannibalVox/cgoparam"
-	"github.com/cockroachdb/errors"
 	"unsafe"
```

Comment:
The reason imports are re-arranged (i.e. `unsafe` moved to the top) is since [goimports](https://pkg.go.dev/golang.org/x/tools/cmd/goimports) was used to sort imports, and it groups standard library imports away from external imports.